gdal2mbtiles/vips.py

```diff
@@ -188,7 +116,11 @@ def set_concurrency(self, processes):
 VIPS = LibVips()
 
 
-class VImage(vipsCC.VImage.VImage):
```

Comment:
Trying to wrap my head around this new vips library here, so bear with me; what's the rationale for not inheriting from `pyvips.Image` (the pyvips equivalent of `vipsCC.Vimage.VImage` AFAICT)?